vertical-pod-autoscaler/RELEASE.md

```diff
@@ -0,0 +1,159 @@
+# VPA Release Instructions
+
+These are instructions for releasing VPA. We aim to release a new VPA minor version after each minor Kubernetes release.
+We release patch versions as needed.
+
+Before doing the release for the first time check if you have all the necessary permissions.
+
+There are the following steps of the release process:
+
+1. [ ] Open issue to track the release.
+2. [ ] Update VPA version const.
+3. [ ] Build and stage images.
+4. [ ] Test the release.
+5. [ ] Promote image.
+6. [ ] Finalize release.
+
+## Open issue to track the release
+
+We use the issue to communicate what is state of the release.
```

Comment:
Do we have a template for the release tracking issue?
If not, maybe we can add it together with the next release or even inside this PR?

Reply:
No, we don't have a template. For now I've only opened an issue, called it something like "Release VPA 0.X.X" and linked all info that seemed relevant (for example #5355).

I opened #5454 to add a template